kong/plugins/prometheus/exporter.lua

```diff
@@ -141,6 +141,17 @@ local function init()
                                                      {"node_id", "shared_dict", "kong_subsystem"},
                                                      prometheus.LOCAL_STORAGE)
 
+  if kong.configuration.database == "off" then
+    memory_stats.lmdb = prometheus:gauge("memory_lua_lmdb_bytes",
+                                         "Allocated bytes in LMDB",
+                                         {"node_id", "kong_subsystem"},
+                                         prometheus.LOCAL_STORAGE)
+    memory_stats.lmdb_capacity = prometheus:gauge("memory_lua_lmdb_total_bytes",
+                                                  "Total capacity in bytes of LMDB",
+                                                  {"node_id", "kong_subsystem"},
```

Comment:
The "kong_subssytem" label is perhaps not required since there is a single LMDB map across HTTP and Stream subsystems.